Description: Fix manpage.
 Avoid spelling-error-in-manpage and hyphen-used-as-minus-sign Lintian warnings.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-10-23

--- a/wmifs/wmifs.1
+++ b/wmifs/wmifs.1
@@ -12,12 +12,12 @@
 wmifs \- a dockable network traffic monitor
 .SH SYNOPSIS
 .B wmifs
-[-d <display name>] [-i <interface name>] [-l] [-w] [-h] [-v]
+[\-d <display name>] [\-i <interface name>] [\-l] [\-w] [\-h] [\-v]
 
 .SH DESCRIPTION
 WMiFS is a dockable applet for X11 that can monitor all your network
 interfaces. It's designed for WindowMaker's Dock and AfterStep's Wharf,
-but it's not dependant on these window managers and should work with
+but it's not dependent on these window managers and should work with
 any other.
 It can display the net load using a standard bar graph or using a waveform
 mode. It also displays the state of the interface with a group of leds.
